lighthouse-core/report/report-generator.js

```diff
@@ -103,7 +103,10 @@ class ReportGenerator {
 
     // Convert numbers to fixed point decimals
     Handlebars.registerHelper('decimal', number => {
-      return number.toFixed(2);
+      if (number && number.toFixed) {
```

Comment:
Ah, I was referring to lighthouse's bugs, not chrome's. 

The root bug for this particular thing appears to be an unexpected order of trace events, which makes it difficult to calculate the navigationStart time. This ends up with a few auditResult values as undefined, null, or NaN, and obviously `.toFixed` fails on those.

While I want to fix the core issue here, I also think report generation should be a bit more resilient when unexpected data comes over.